_infra/helm/ras-rm-benchmark/templates/job.yaml

```diff
@@ -3,7 +3,7 @@ kind: Job
 metadata:
   name: {{ .Chart.Name}}
 spec:
-  ttlSecondsAfterFinished: 0
+  ttlSecondsAfterFinished: 3600
```

Comment:
If the job is removed when it completes, the Spinnaker step will never go green (it times-out and goes red !). Leaving the completed job in the Cluster means Spinnaker can see its completed successfully and the step goes green.